messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistSdk.kt

```diff
@@ -251,6 +257,14 @@ object GistSdk : Application.ActivityLifecycleCallbacks {
         return sharedPreferences.getString(SHARED_PREFERENCES_USER_TOKEN_KEY, null)
     }
 
+    private fun forEachListener(action: (MutableIterator<GistListener>) -> Unit) {
```

Comment:
lets add a unit test for this method?

Reply:
Yeah, I'm writing down some tests for this, will push once I'm able to fail it for the current code and pass for the new changes.